core/org.wso2.carbon.utils/src/main/java/org/wso2/carbon/CarbonConstants.java

```diff
@@ -449,68 +449,6 @@ public static class LogEventConstants {
         public static final String RECORDED_AT = "recordedAt";
         public static final String CLIENT_COMPONENT = "clientComponent";
         public static final String CORRELATION_ID = "correlationId";
-
-        // keys for initiator details in audit logs.
-        public static final String INITIATOR_ID = "initiator.id";
```

Comment:
Removing the constants that are not used in audit logs

Reply:
@AnuradhaSK Can you please confirm whether we have used these anywhere?

Reply:
This is intended to be used only in the new patterned audit logs. Since we are changing them, there shouldn't be other usage